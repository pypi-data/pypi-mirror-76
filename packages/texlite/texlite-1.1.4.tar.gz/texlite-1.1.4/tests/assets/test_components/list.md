Here is an unordered list:
- An item
- Another item
- A third item

This is an unordered list using different prefixes:
+ An item
- Another item
* A third item

Here is an ordered list:
1. My first item
2. My second item
3. My third item
4. And one more, why not?
