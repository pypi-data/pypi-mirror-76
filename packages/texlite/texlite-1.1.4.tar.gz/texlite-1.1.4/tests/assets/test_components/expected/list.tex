% meta
\documentclass[10pt]{extarticle}

% packages
\usepackage[utf8]{inputenc}
\usepackage[margin=1.6in]{geometry}
\usepackage{hyperref}
\usepackage{amsmath}
\usepackage{amssymb}

% preamble commands
\linespread{1.0}
\usepackage{graphicx}
\graphicspath{{tests/assets/test_components/}}

% document details
\date{}

% ------------------------------------------------------------------------------
\begin{document}
% ------------------------------------------------------------------------------

Here is an unordered list:

\begin{itemize}
\item An item
\item Another item
\item A third item
\end{itemize}

This is an unordered list using different prefixes:

\begin{itemize}
\item An item
\item Another item
\item A third item
\end{itemize}

Here is an ordered list:

\begin{enumerate}
\item My first item
\item My second item
\item My third item
\item And one more, why not?
\end{enumerate}

% ------------------------------------------------------------------------------
\end{document}
% ------------------------------------------------------------------------------

